packages/oui-datagrid/src/cell/cell.controller.js

```diff
@@ -19,6 +23,10 @@ export default class {
         } else {
             this._compileCell();
         }
+
+        this.cellScope.$watch(() => this.datagridCtrl.selectedRows[this.index], (isSelected) => {
+            this.cellScope.$isSelected = isSelected || false;
```

Comment:
`this.cellScope.$isSelected = !!isSelected` is possible no ?

Reply:
`!!` is harder to read by someone not used to this js "trick" but yes it would fit the job

Reply:
Ok :)